.github/workflows/build-publish.yaml

```diff
@@ -35,27 +42,43 @@ jobs:
         id: meta
         uses: docker/metadata-action@v5
         with:
-          images: ${{ github.repository }}
+          images: 
+            ${{ github.repository }}
+            ${{ env.GAR_LOCATION }}-docker.pkg.dev/${{ env.GCP_PROJECT_ID }}/${{ env.GAR_REPOSITORY }}/${{ env.IMAGE }}
           # https://github.com/marketplace/actions/docker-metadata-action#tags-input
           tags: |
             type=semver,pattern={{raw}}
             type=raw,value=latest,enable={{is_default_branch}}
             type=sha,prefix=,enable={{is_default_branch}}
```

Comment:
Don't we set tag here? https://github.com/mozilla-services/telescope/blob/ba8b8ac7914a039db41c8ca3436dbba3d65f2ec7/.github/workflows/build-publish.yaml#L56C18-L58C29

Reply:
This tagging strategy matches what we do with Dockerhub, since I only added another image repository. Happy to add the `JBI_TAG` tag here to keep things consistent with other syseng projects, though

Reply:
I'm going to merge this as-is, and if we want to revert back to `git describe --tags --abbrev=4` tags, we can do so 👍🏻